frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/FetchAllRemotesAction.java

```diff
@@ -33,9 +33,12 @@ public void update(AnActionEvent anActionEvent) {
     super.update(anActionEvent);
 
     var project = getProject(anActionEvent);
+    var presentation = anActionEvent.getPresentation();
```

Comment:
Hmmm wait... 1) is this change the entire fix for #321? 2) if so, how does it work? :thinking:

Reply:
yup, the description had been changed while running fetch but has not been changed after that back to the correct description. that's all

Reply:
mutable objects

![image](https://user-images.githubusercontent.com/19799111/83772594-d4d6e080-a683-11ea-8a37-55540e2ba040.png)

Reply:
Oh ok, I didn't notice the newly-added `else` ;)